### ISSUE TITLE
UX: code display does not scroll right far enough

### ISSUE THREAD

**Issue by sygibson** [digitalrebar/provision#1250]
Examining a BootEnv with a `Description` field that has a long line of text causes the code display to not render all of the text to the right.

example:
![Screen Shot 2019-06-11 at 11 02 42](https://user-images.githubusercontent.com/5308433/59295466-a113d700-8c38-11e9-917e-ce3bb66a02ce.png)

This makes it impossible to view the full JSON code for long lines.

**Comment by Meshiest** [comment]
https://stackoverflow.com/questions/38323427/remove-word-wrap-offset-in-ace-editor

Would this help @sygibson ?

**Comment by sygibson** [comment]
I don't think that will help - we currently do not line wrap. It appears that the length of the line has a limitation we're exceeding.  Line wrapping might help, but may also cause confusing results too.  If there is someway to highlight that the line is wrapped - I see that they're is a "softwrap" feature, which may visually work - not sure.
